edtf/templates/edtf/edtf_form.html

```diff
@@ -43,11 +43,15 @@ <h1>
     "validEDTF": true,
 })
 
```

Comment:
There is another EDTF spec link in the intro above that should be updated to https://www.loc.gov/standards/datetime/.